.github/workflows/system-tests-official.yml

```diff
@@ -0,0 +1,129 @@
+name: System Tests - Official
+on: # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      # A workaround to trigger the workflow for pull requests from forked repository,
+      # which does not have access to secrets.
+      #
+      # This is also useful for testing the workflow without opening a pull request.
+      - tmp/*
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch: {}
+  schedule:
+    - cron: "00 04 * * 2-6"
+
+# Default permissions for all jobs
+permissions: {}
+
+jobs:
+  changes:
+    name: Changes
+    # temporary condition to test official workflow
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.head_ref == 'cbeauchesne/system-tests-official')
```

Comment:
`github.head_ref` condition should probably be removed.

Reply:
I need to keep it during the review, if some comments needs changes.

Even if it does not hurst, I can remove it just before the merge, as you wish.

Reply:
Addressed in [5a8cc4f](https://github.com/DataDog/dd-trace-rb/pull/4774/commits/5a8cc4f9bb6db69e4311d460bebb15410c7fd932)